src/core/Core.js

```diff
@@ -477,7 +477,7 @@ class Uppy {
   retryAll () {
     const updatedFiles = Object.assign({}, this.getState().files)
     const filesToRetry = Object.keys(updatedFiles).filter(file => {
-      return updatedFiles[file].error
+      return updatedFiles[file].error.message
```

Comment:
Should we be checking if `error.message` exists first? What if string is returned from somewhere? Though I think we have an Eslint rule for that now, an extra check would not be too bad?

Reply:
I think we don't need to add the `.message` there at all, because it's inside a `filter`. It can just return `updatedFiles[file].error` here.